models/createComment.js

```diff
@@ -0,0 +1,27 @@
+const db = require("../db/connection");
+const format = require("pg-format");
+
+const createComment = (req, articleID) => {
+  const username = req.username;
+  const body = req.body;
+  const commentQuery = `INSERT INTO comments (author, body, article_id) VALUES ($1, $2, $3) RETURNING *;`;
+  const articleQuery = format(
+    `SELECT * FROM articles WHERE article_id = %L;`,
+    articleID
+  );
+  return db.query(articleQuery).then((results) => {
+    if (results.rows.length === 0) {
+      const err = new Error("Article not found");
```

Comment:
article_id is also a foreign key so again, a separate check that the article exists should not be necessary as a psql error will be thrown